.flake8

```diff
@@ -0,0 +1,42 @@
+[flake8]
+ignore = 
+    # There's nothing wrong with assigning lambdas
+    E731,
+    # PEP8 weakly recommends Knuth-style line breaks before binary
+    # operators
+    W503
+exclude = 
+    # These are directories that it's a waste of time to traverse
+    .git,
+    .tox,
+    ansible,
+    bin,
+    config,
+    docs,
+    gulp,
+    picard_scripts,
+    requirements,
+    node_modules,
+    site,
+
+    # And directories in cfgov that don't have python files to lint
+    cfgov/es_mappings,
+    cfgov/jinja2,
+    cfgov/static_built,
+    cfgov/templates,
+    cfgov/unprocessed,
+
+    # Generated migration files will throw errors. We need to find a way
+    # to exclude django-generated migrations while including
+    # manually-written migrations.
+    cfgov/*/migrations/*,
```

Comment:
Hmm, good point, I am going to play around with it a little bit. That regex didn't work, unfortunately.